webpack.config.js

```diff
@@ -209,7 +245,19 @@ const webviewConfig = {
         ]
       }
     ]
-  }
+  },
+  plugins: [
+    new ContextMapPlugin(
+      'node_modules/context-eval',
+      ['./lib/context-node']
+    ),
+    new webpack.ProvidePlugin({
+      Buffer: ['buffer', 'Buffer'],
+    }),
+    new webpack.ProvidePlugin({
+      process: 'process/browser'
+    }),
```

Comment:
Tiny nit, feel free to ignore: you can do it in one plugin

```suggestion
    new webpack.ProvidePlugin({
      Buffer: ['buffer', 'Buffer'],
      process: 'process/browser'
    }),
```